simulator/simple_planning_simulator/src/simple_planning_simulator/vehicle_model/sim_model_ideal_steer_acc_geared.cpp

```diff
@@ -62,19 +62,24 @@ Eigen::VectorXd SimModelIdealSteerAccGeared::calcModel(
 float64_t SimModelIdealSteerAccGeared::calcVelocityWithGear(
   const Eigen::VectorXd & state, const uint8_t gear) const
 {
-  using autoware_auto_vehicle_msgs::msg::VehicleStateCommand;
-  if (gear == VehicleStateCommand::GEAR_DRIVE ||
-    gear == VehicleStateCommand::GEAR_LOW ||
-    gear == VehicleStateCommand::GEAR_NEUTRAL)
-  {
+  using autoware_auto_vehicle_msgs::msg::GearCommand;
+  if (
+    gear == GearCommand::DRIVE || gear == GearCommand::DRIVE_2 || gear == GearCommand::DRIVE_3 ||
+    gear == GearCommand::DRIVE_4 || gear == GearCommand::DRIVE_5 || gear == GearCommand::DRIVE_6 ||
+    gear == GearCommand::DRIVE_7 || gear == GearCommand::DRIVE_8 || gear == GearCommand::DRIVE_9 ||
+    gear == GearCommand::DRIVE_10 || gear == GearCommand::DRIVE_11 ||
+    gear == GearCommand::DRIVE_12 || gear == GearCommand::DRIVE_13 ||
+    gear == GearCommand::DRIVE_14 || gear == GearCommand::DRIVE_15 ||
+    gear == GearCommand::DRIVE_16 || gear == GearCommand::DRIVE_17 ||
+    gear == GearCommand::DRIVE_18 || gear == GearCommand::LOW || gear == GearCommand::LOW_2) {
```

Comment:
maybe we don't have to care about GearCommand::DRIVE_* and GearCommand::REVERSE_* ? @TakaHoribe

Reply:
Let's use the same definition used [here](https://github.com/tier4/autoware_iv_msgs/blob/use-autoware-auto-msgs/autoware_iv_auto_msgs_converter/include/autoware_iv_auto_msgs_converter/autoware_iv_auto_msgs_converter.hpp#L147-L189).

So it looks ok.

Reply:
Then should I fix it here as well?
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/62e653afbbb36a9c5471a1cd11728d6f5dc6b73a/vehicle/as/src/pacmod_interface/pacmod_interface.cpp#L532-L551

Reply:
I discussed with Kimura-san. pacmod_interface only supports pacmod, which is supposed to send DRIVE, REVERSE, PARK ... (I mean, not DRIVE_2, 3, ...). So we don't have to modify pacmod_interface.

Reply:
I agree. pacmod_interface is vehicle-specific, so you don't have to support all message types.